services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/web/SwitchManagerWebRoutable.java

```diff
@@ -33,7 +33,7 @@ public Restlet getRestlet(Context context) {
         router.attach("/flow", FlowResource.class);
         router.attach("/flows/switch_id/{switch_id}", FlowsResource.class);
         router.attach("/meters/switch_id/{switch_id}", MetersResource.class);
-
+        router.attach("/switch/enable_bfd", EnableBfdResource.class);
```

Comment:
What if later we need to disable bfd? "/switch/{switch_id}/bfd" ?

Reply:
We have "complete" kafka API. This REST API is some set of back door required for initial system setup. I hope it will be dropped soon.